targets/python/templates/PlayFabUtil.py.ejs

```diff
@@ -0,0 +1,59 @@
+# using PlayFab.Json;
+import json
+<% if (hasClientOptions) {
+%>import PlayFabClientModels
+<% } else if (hasServerOptions) {
+%>import PlayFabServerModels
+<% } %>
+
+class PlayFab:
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
+    class Unordered(attribute):
+        def __init__():
+            pass
+
+        def __init__(self, sortProperty):
+            self.SortProperty = sortProperty;
+
+    class PlayFabUtil:
+        # All parseable ISO 8601 formats for DateTime.[Try]ParseExact - Lets us deserialize any legacy timestamps in one of these formats
+        # (index 0-4) These are the standard format with ISO 8601 UTC markers (T/Z)
+        # (index 5-9) These are the standard format without ISO 8601 UTC markers (T/Z)
+        DefaultDateTimeFormats = ["yyyy-MM-ddTHH:mm:ss.FFFFFFZ", "yyyy-MM-ddTHH:mm:ss.FFFFZ", "yyyy-MM-ddTHH:mm:ss.FFFZ", "yyyy-MM-ddTHH:mm:ss.FFZ", "yyyy-MM-ddTHH:mm:ssZ", "yyyy-MM-dd HH:mm:ss.FFFFFF", "yyyy-MM-dd HH:mm:ss.FFFF", "yyyy-MM-dd HH:mm:ss.FFF", "yyyy-MM-dd HH:mm:ss.FF", "yyyy-MM-dd HH:mm:ss"]
+
+        DEFAULT_UTC_OUTPUT_INDEX = 2 # The default format everybody should use
+        DEFAULT_LOCAL_OUTPUT_INDEX = 7 # The default format if you want to use local time (This doesn't have universal support in all PlayFab code)
+
+        def GenerateErrorReport(error):
+            if error == None:
+                return None
+            return error.GenerateErrorReport()
+
+<% if (hasClientOptions || hasServerOptions) { %>
+        def GetCloudScriptErrorReport(result):
```

Comment:
Let's not bother with GetCloudScriptErrorReportmethod this pass.
This is a long-term nice-to have feature, but we shouldn't take time testing it in this pass.